.github/workflows/build.yaml

```diff
@@ -5,34 +5,16 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
This workflow has a too broad push trigger (`on`) since it would also build on tag pushes. (I can't mark it further up in the file in the Gh review). however we don't need most of the jobs here again for releases (e.g. linting, tests) since they have run before with the merge of a PR and within PRs.
We usually do triggers like
```yaml
on:
  push:
    branches:
    - master
  pull_request: {} 
```

Reply:
Perfect, thanks!